services/resource-usage-tracker/tests/unit/test_resource_tracker.py

```diff
@@ -0,0 +1,56 @@
+# pylint: disable=no-value-for-parameter
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+import asyncio
+from unittest import mock
+
+import pytest
+from fastapi import FastAPI
+from pytest_mock import MockerFixture
+from pytest_simcore.helpers.utils_envs import EnvVarsDict
+from simcore_service_resource_usage_tracker.core.settings import ApplicationSettings
+
+_FAST_POLL_INTERVAL = 1
+
+
+@pytest.fixture
+def app_environment(
+    app_environment: EnvVarsDict,
+    monkeypatch: pytest.MonkeyPatch,
+) -> EnvVarsDict:
+    # fast interval
```

Comment:
TIP: `setenvs_from_dict` is very convenient for ``app_environment``-like overrides
```python
import pytest
from pytest_simcore.helpers.utils_envs import setenvs_from_dict
from pytest. import MonkeyPatch

@pytest.fixture
def app_environment(
    app_environment: EnvVarsDict,
    monkeypatch: MonkeyPatch,
) -> EnvVarsDict:
   return app_environment | setenvs_from_dict( monkeypatch,
        {
            "RESOURCE_USAGE_TRACKER_EVALUATION_INTERVAL_SEC": f"{_FAST_POLL_INTERVAL}",
        },
    )
```

Reply:
done